externs/olx.js

```diff
@@ -3547,7 +3598,7 @@ olx.layer.VectorOptions.prototype.renderBuffer;
 
 /**
  * Source.
- * @type {ol.source.Vector}
+ * @type {ol.source.Vector|ol.source.VectorTile}
```

Comment:
so there is a possible use case for combining ol.layer.Vector and ol.source.VectorTile?